modules/__tests__/Media-ssr-test.js

```diff
@@ -15,15 +16,15 @@ describe("A <Media> in server environment", () => {
     it("should render its children as if all queries are matching", () => {
       const element = (
         <Media queries={queries}>
-          {matches =>
-            matches.sm &&
-            matches.lg &&
-            matches.xl && <span>All matches, render!</span>
-          }
+          {matches => (
```

Comment:
Change. This was failing because of things I missed during my first pass, and it was erroring because it wasn't returning null.